docs/INSTALLATION.md

```diff
@@ -1,13 +1,15 @@
 ## Dart
-* [Install Dart](https://www.dartlang.org/install)
-* Add the Dart SDK to your path.
-* From a new command window, type `dart --version` to verify that Dart is now available.
+* One option is to follow the [instructions for installation](https://www.dartlang.org/install) on dartlang.org.
 
-## Dart Plugin for your IDE or Editor
-* If you are using an IDE or an editor, [install and configure the Dart plugin](https://www.dartlang.org/tools).
+### Manual install instruction
+* Download a zip file from [this page](https://www.dartlang.org/install/archive).
+* Unzip the file into whatever directory you choose.
+* Add the full path of that directory to your PATH.
+    * Follow the links below for updating your PATH on Windows, Linux and Mac platforms:
+        * [Instructions for Windows](https://www.computerhope.com/issues/ch000549.htm)
+        * [Instructions for Linux](https://www.computerhope.com/issues/ch001647.htm)
+        * [Instructions for Mac](https://www.pegaxchange.com/tag/macos-sierra-add-entry-to-path-variable/)
+* From a new command window or terminal, type `dart --version` to verify that Dart is now available.
```

Comment:
To make it clearer, and if you think it makes sense, maybe add another sentence like, "If the output shows the version number, then Dart is correctly installed on your computer."